src/interactive/repl.ts

```diff
@@ -376,6 +376,97 @@ function clearProgress() {
     }
 }
 
+function display(params: { kind: string, data: any }) {
+    if (params.kind === 'application/vnd.julia-vscode.diagnostics') {
+        displayDiagnostics(params)
+    } else {
+        plots.displayPlot(params)
+    }
+}
+
+interface diagnosticData {
+    msg: string,
+    path: string,
```

Comment:
Should this maybe be a `uri`? The entire VS Code ecosystem seems to use Uris everywhere now, and that might make it easier down the road to for example designate locations in notebooks?

Reply:
We can't, unless Julia is also URI aware (which it isn't). So yes, possible that this is something we need to tackle in the future, but definitely not here.